frontend/packages/design-system/src/types/icons.ts

```diff
@@ -1,14 +1,8 @@
 export type IconProps = {
```

Comment:
Is this still used?

Reply:
this one not, I'll move the updated type here though